tests/functional/adapter/test_partitions.py

```diff
@@ -264,3 +286,35 @@ def test__check_run_with_partitions(self, project):
         records_count_first_run = project.run_sql(model_run_result_row_count_query, fetch="all")[0][0]
 
         assert records_count_first_run == 202
+
+
+class TestIcebergTablePartitionsBuckets:
```

Comment:
Could you add yet another test where you avoid to combine partitioning and bucketing and use only bucketing instead? So pretty much what I'm asking is to add another test where you use this
`"+partitioned_by": ["bucket(non_random_str, 5)"],`

Reply:
Added ✅